src/io/flutter/utils/FlutterModuleUtils.java

```diff
@@ -59,21 +59,24 @@ public static boolean hasFlutterModule(@NotNull Project project) {
     // If not IntelliJ, assume a small IDE (no multi-module project support).
     // Look for a module with a flutter-like file structure.
     if (!PlatformUtils.isIntelliJ()) {
-      for (Module module : ModuleManager.getInstance(project).getModules()) {
-        if (usesFlutter(module)) {
-          return true;
-        }
+      if (CollectionUtils.anyMatch(getModules(project), FlutterModuleUtils::usesFlutter)) {
```

Comment:
I'm not entirely in love with java8's flavor of terseness and while I wouldn't argue for being prescriptive, in some cases I think things are made clearer.  For example,

```CollectionUtils.filter(getModules(project), m -> isFlutterModule(m) || usesFlutter(m))```

is I think nicer than it's alternative(s).

But anyway, I think you're right this is a style thing and I promise not to troll your PRs with suggestions to use these idioms!